kolibri/plugins/coach/assets/src/state/actions/reports.js

```diff
@@ -401,19 +400,21 @@ export function setReportSorting(store, { sortColumn, sortOrder }) {
 export function showRecentItemsForChannel(store, params) {
   const { classId, channelId } = params;
   preparePageNameAndTitle(store, PageNames.RECENT_ITEMS_FOR_CHANNEL);
-  const channelPromise = ChannelResource.getModel(channelId).fetch();
+  const channelPromise = ChannelResource.fetchModel({ id: channelId });
 
   Promise.all([channelPromise, setClassState(store, classId)]).then(
     ([channelData]) => {
       const threshold = now();
       threshold.setDate(threshold.getDate() - RECENCY_THRESHOLD_IN_DAYS);
-      const recentReportsPromise = RecentReportResource.getCollection({
-        channel_id: channelId,
-        content_node_id: channelData.root,
-        collection_kind: UserScopes.CLASSROOM,
-        collection_id: classId,
-        last_active_time: threshold,
-      }).fetch();
+      const recentReportsPromise = RecentReportResource.fetchCollection({
+        getParams: {
+          channel_id: channelId,
+          content_node_id: channelData.root,
+          collection_kind: UserScopes.CLASSROOM,
+          collection_id: classId,
+          last_active_time: threshold,
```

Comment:
@rtibbles I was getting errors sending the long-form date string to the server as a filter param. I fixed it by converting to IsoString. Is that the right fix?

```diff
-          last_active_time: threshold,
+          last_active_time: threshold.toISOString(),
```

Reply:
Seems right, before this refactor, the threshold was simply being ignored, so no error was being produced.